wiki/People/Users_with_unique_titles/id.md

```diff
@@ -1,18 +1,14 @@
----
-outdated: true
----
+# Daftar pengguna dengan gelar khusus
 
-# Pengguna dengan gelar unik
+Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [gugus pengguna (*user group*)](/wiki/People/The_Team#user-groups). Meskipun demikian, terdapat beberapa pengguna yang secara khusus dianugerahi gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang telah mereka torehkan.
```

Comment:
```suggestion
Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [grup tim (*user group*)](/wiki/People/The_Team#user-groups). Meskipun demikian, terdapat beberapa pengguna yang secara khusus diberikan gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang pernah mereka lakukan.
```

Reply:
it should redirect to group member instead. so `gugus pengguna` is correct here.

Reply:
ya famoss is correct here; the current sentence is correct as is and changing it to `grup tim` would make it misleading instead